jetstream/validation.py

```diff
@@ -0,0 +1,80 @@
+"""Validation helpers."""
+
+import os
+import toml
+from datetime import datetime
+from pathlib import Path
+
+from .analysis import Analysis
+from .config import AnalysisSpec, OutcomeSpec, PLATFORM_CONFIGS
+from .dryrun import DryRunFailedError
+from .experimenter import Experiment, ExperimentCollection
+from .external_config import OUTCOMES_DIR
+
+
+def validate_config(path: str) -> bool:
+    """Validate all config files in the provided path."""
+    config_files = [p for p in path if os.path.isfile(p)]
+
+    collection = ExperimentCollection.from_experimenter()
+
+    for file in config_files:
+        print(f"Validate {file}")
+
+        if Path(file).parent.parent.name == OUTCOMES_DIR:
+            # file is an outcome snippet
+            outcomes_spec = OutcomeSpec.from_dict(toml.load(file))
+            platform = Path(file).parent.name
+
+            if platform not in PLATFORM_CONFIGS:
+                print(f"Platform '{platform}' is unsupported.")
+                return False
+            else:
+                if platform == "firefox_desktop":
+                    app_id = "firefox-desktop"
+                else:
+                    app_id = "org.mozilla.fenix"
```

Comment:
The alternative to hardcoding the app_ids here, could also be adding a list of supported `applications` to the `Platform` object and look them up here.

Reply:
Adding them to the Platform object sounds like a great idea.